css-two-column-layout/styles.css

```diff
@@ -0,0 +1,83 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: auto;
+}
+
+.column-full {
+  text-align: center;
+  max-width: 650px;
+  margin: auto;
+}
+
+h1 {
+  letter-spacing: 0.1em;
+  font-size: 45px;
+}
+
+h1,
+h3 {
+  text-transform: uppercase;
+  font-family: 'Karla', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+}
+
+p {
+  font-family: 'Karla', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  color: #6a747d;
+  font-size: 15px;
+  line-height: 200%;
+}
+
+.container {
+  display: flex;
+  width: 1100px;
+  flex-wrap: wrap;
+  margin: auto;
+  padding-top: 30px;
+}
+
+.row {
+  display: flex;
+  justify-content: center;
+  width: 45%;
```

Comment:
A `.row` being set to a width other than `100%` is unusual. With the way you built your HTML, you should have one `.row`, which contains two `.column-half` elements, which has two `.content-holder` boxes vertically stacked. This will allow the row to be the full width, which is the correct path to take in a grid system.

Reply:
![Screen Shot 2021-01-23 at 2 40 00 PM](https://user-images.githubusercontent.com/74885918/105616070-f7d81100-5d88-11eb-9d12-165f355e6065.png)